scripts/pre-commit-check-code-generation

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+GO111MODULE=off go get golang.org/x/tools/cmd/goimports
+go generate ./...
+if [[ -n ${CIRCLECI-} ]]; then
+  ~/go/bin/goimports -w -local github.com/cmsgov/easi-app pkg/graph/schema.resolvers.go
+else
```

Comment:
Attempting to simply run `goimports` in CI generated this error: `goimports: command not found`. Our current CI image doesn't have `GOPATH` set, so this is a bit of a workaround. Again...there may be a better way to do this.